src/components/Main.tsx

```diff
@@ -20,26 +31,43 @@ const filters = [
 
 const Main: React.FC = () => {
     const [select, setSelect] = useState(filters[0].value)
+    const isOpenForm = useSelector(selectOpenForm)
+    const isEdit = useSelector(selectEditStatus)
+    const isNotEmptyListMovies = useSelector(selectNotEmptyListMovies)
+
     const dispatch = useDispatch()
-    
+
+    const closeFormModal = () => {
+        dispatch(setOpenForm(false))
+    }
+
     const sort = (value) => dispatch(sortMovies(value))
 
     const listFilters = filters.map((filter) => {
-        return <option 
-                key={filter.key} 
+        return <option
+                key={filter.key}
                 value={filter.value}
             >
                 {filter.value.toUpperCase()}
             </option>
     })
-    
+
     const handleSelect = ({ target }) => {
         const { value } = target
         setSelect(value)
         const filter = filters.find((item) => item.value == value)
         if(filter) sort(filter.key)
     }
 
+    let query = useQuery();
```

Comment:
it should be `const` i suppose and it will be move to the beginning

Reply:
changed